web/app/js/components/TopRoutes.jsx

```diff
@@ -39,9 +37,10 @@ const topRoutesQueryProps = {
 };
 const topRoutesQueryPropType = PropTypes.shape(topRoutesQueryProps);
 
-const urlPropsQueryConfig = _mapValues(topRoutesQueryProps, () => {
-  return { type: UrlQueryParamTypes.string };
-});
+let topRoutesQueryConfig = {};
```

Comment:
Here you are defining the variable with `let`, but line 48 of `TapQueryForm` uses the same logic with `const`. Even though object keys are not protected by `const` and therefore can be changed/added/updated, I think it's better to define with `let` in both cases, for consistency and to be clear that the variable values will change.